plugins/language_julia.lua

```diff
@@ -4,109 +4,130 @@
+    ["Union"]        = "keyword",
+    ["else"]         = "keyword",
+    ["elseif"]       = "keyword",
+    ["end"]          = "keyword",
+    ["export"]       = "keyword",
+    ["finally"]      = "keyword",
+    ["for"]          = "keyword",
+    ["function"]     = "keyword",
+    ["global"]       = "keyword",
+    ["if"]           = "keyword",
+    ["in"]           = "keyword",
+    ["import"]       = "keyword",
+    ["let"]          = "keyword",
+    ["local"]        = "keyword",
+    ["macro"]        = "keyword",
+    ["type"]         = "keyword",
+    ["module"]       = "keyword",
+    ["mutable"]      = "keyword",
+    ["quote"]        = "keyword",
+    ["return"]       = "keyword",
+    ["try"]          = "keyword",
+    ["typeof"]       = "keyword",
+    ["using"]        = "keyword",
+    ["while"]        = "keyword",
+    ["where"]        = "keyword",
 
     -- types
-    ["struct"]            = "keyword2",
-    ["abstract"]          = "keyword2",
-    ["primitive"]         = "keyword2",
-    ["mutable"]           = "keyword2",
-    ["Char"]              = "keyword2",
-    ["Bool"]              = "keyword2",
-    ["Int"]               = "keyword2",
-    ["Integer"]           = "keyword2",
-    ["Int8"]              = "keyword2",
-    ["UInt8"]             = "keyword2",
-    ["Int16"]             = "keyword2",
-    ["UInt16"]            = "keyword2",
-    ["Int32"]             = "keyword2",
-    ["UInt32"]            = "keyword2",
-    ["Int64"]             = "keyword2",
-    ["UInt64"]            = "keyword2",
-    ["Int128"]            = "keyword2",
-    ["UInt128"]           = "keyword2",
-    ["Float16"]           = "keyword2",
-    ["Float32"]           = "keyword2",
-    ["Float64"]           = "keyword2",
-    ["Vector"]            = "keyword2",
-    ["Matrix"]            = "keyword2",
-    ["Ref"]               = "keyword2",
-    ["String"]            = "keyword2",
-    ["Function"]          = "keyword2",
-    ["Number"]            = "keyword2",
+    ["struct"]       = "keyword2",
+    ["abstract"]     = "keyword2",
+    ["primitive"]    = "keyword2",
+    ["Char"]         = "keyword2",
+    ["Bool"]         = "keyword2",
+    ["Int"]          = "keyword2",
+    ["Integer"]      = "keyword2",
+    ["Int8"]         = "keyword2",
+    ["UInt8"]        = "keyword2",
+    ["Int16"]        = "keyword2",
+    ["UInt16"]       = "keyword2",
+    ["Int32"]        = "keyword2",
+    ["UInt32"]       = "keyword2",
+    ["Int64"]        = "keyword2",
+    ["UInt64"]       = "keyword2",
+    ["Int128"]       = "keyword2",
+    ["UInt128"]      = "keyword2",
+    ["Float16"]      = "keyword2",
+    ["Float32"]      = "keyword2",
+    ["Float64"]      = "keyword2",
+    ["Vector"]       = "keyword2",
+    ["Matrix"]       = "keyword2",
+    ["Ref"]          = "keyword2",
+    ["String"]       = "keyword2",
+    ["Function"]     = "keyword2",
+    ["Number"]       = "keyword2",
+    ["im"]           = "keyword2",
```

Comment:
Is this a keyword?

Reply:
I'm not sure, it was like that before, so I just left it there. 

I can change it though